### ISSUE TITLE
readProfiles() failing on indexSynthetic

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#111]
```R
library(argoFloats)
data("indexSynthetic") # 39 items
p <- getProfiles(indexSynthetic)
a <- readProfiles(p)
```
yields
```R
R version 4.0.0 (2020-04-24) -- "Arbor Day"
...
> library(argoFloats)
Registered S3 methods overwritten by 'adehabitatMA':
  method                       from
  print.SpatialPixelsDataFrame sp  
  print.SpatialPixels          sp  
> data("indexSynthetic") # 39 items
> p <- getProfiles(indexSynthetic)
> a <- readProfiles(p)
Error in sort.int(x, na.last = na.last, decreasing = decreasing, ...) : 
  'x' must be atomic
Calls: readProfiles -> unique -> sort -> sort.default -> sort.int
Execution halted
```

**Comment by dankelley** [comment]
I get the same error with
```R
data(indexMerged)
```

**Comment by dankelley** [comment]
This is solved in "develop" commit 34f08c67a200ffd03e5a38f12203ba5278e12ea1
